project/Http4sPlugin.scala

```diff
@@ -31,11 +31,11 @@ object Http4sPlugin extends AutoPlugin {
     ThisBuild / http4sApiVersion := (ThisBuild / version).map {
       case VersionNumber(Seq(major, minor, _*), _, _) => (major.toInt, minor.toInt)
     }.value,
-  )
+  ) ++ sbtghactionsSettings
 
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     scalaVersion := scala_213,
-    crossScalaVersions := Seq(scala_213, scala_212),
+    ThisBuild / crossScalaVersions := Seq(scala_213, scala_212),
```

Comment:
Can a `ThisBuild` go in `projectSettings`?  I thought that needed to be in `buildSettings`?

Reply:
I have only a very rudimentary understanding of the differences between `projectSettings` and `buildSettings`. I had to add `ThisBuild / ` for sbt-github-actions to pick it up, and it appears to work(?). I can move it to though, it seems to continue to work locally with it being moved.